temporian/implementation/numpy/data/io.py

```diff
@@ -122,9 +124,11 @@ def event_set(
         )
 
     # Convert timestamps to expected type.
+    logging.debug("Normalize timestamps")
     timestamps, auto_is_unix_timestamp = normalize_timestamps(timestamps)
 
     if not np.all(timestamps[:-1] <= timestamps[1:]):
+        logging.debug("Sort timestamps")
```

Comment:
would change to Normalizing, Normalizing, Sorting, in all of them

Reply:
*Done